src/modules/auth/googleauth/google.service.ts

```diff
@@ -0,0 +1,34 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, VerifyCallback } from 'passport-google-oauth20';
+import { Injectable } from '@nestjs/common';
+import { Prof } from '@/modules/auth/googleauth/dto/profile.dto';
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
+  constructor() {
+    super({
+      clientID: `${process.env.GOOGLE_CLIENT_ID}`,
+      clientSecret: `${process.env.GOOGLE_CLIENT_SECRET}`,
+      callbackURL: `http://${process.env.HOST}:${process.env.PORT}/auth/google/redirect`,
```

Comment:
full path should be in the env, today you have HTTP, tomorrow you will have https, you approach is not righ

Reply:
fixed